app/routes/book_routes.py

```diff
@@ -49,6 +49,17 @@ def get_one_book(book_id):
         "description": book.description,
     }
 
+@books_bp.put("/<book_id>")
+def update_book(book_id):
+    book = validate_book(book_id)
+    request_body = request.get_json()
+
+    book.title = request_body.get("title")
+    book.description = request_body.get("description")
+    db.session.commit()
+
+    return Response(status=204)
```

Comment:
My initial response if being a little offput by sending an empty dictionary instead of truly no content if that is the goal. I'm wondering if there's a reason `make_response({}, 204)` would be preferable to `Response(status=204, mimetype="application/json")`?

Reply:
Ha ha. Yeah. That's just enough to trigger the internal jsonify. I'm fine with adding the `mimetype` parameter, though.